libr/include/r_io.h

```diff
@@ -146,28 +146,28 @@ typedef struct r_io_plugin_t {
 	const char *license;
 	void *widget;
 	const char *uris;
-	int (*listener)(RIODesc *io);
-	int (*init)(void);
+	int(*listener)(RIODesc *io);
+	int(*init)(void);
 	RIOUndo undo;
 	bool isdbg;
-	// int (*is_file_opened)(RIO *io, RIODesc *fd, const char *);
+	// int(*is_file_opened)(RIO *io, RIODesc *fd, const char *);
 	char *(*system)(RIO *io, RIODesc *fd, const char *);
-	RIODesc* (*open)(RIO *io, const char *, int perm, int mode);
-	RList* /*RIODesc* */ (*open_many)(RIO *io, const char *, int perm, int mode);
-	int (*read)(RIO *io, RIODesc *fd, ut8 *buf, int count);
-	ut64 (*lseek)(RIO *io, RIODesc *fd, ut64 offset, int whence);
-	int (*write)(RIO *io, RIODesc *fd, const ut8 *buf, int count);
-	int (*close)(RIODesc *desc);
-	bool (*is_blockdevice)(RIODesc *desc);
-	bool (*is_chardevice)(RIODesc *desc);
-	int (*getpid)(RIODesc *desc);
-	int (*gettid)(RIODesc *desc);
-	bool (*getbase)(RIODesc *desc, ut64 *base);
-	bool (*resize)(RIO *io, RIODesc *fd, ut64 size);
-	int (*extend)(RIO *io, RIODesc *fd, ut64 size);
-	bool (*accept)(RIO *io, RIODesc *desc, int fd);
-	int (*create)(RIO *io, const char *file, int mode, int type);
-	bool (*check)(RIO *io, const char *, bool many);
+	RIODesc*(*open)(RIO *io, const char *, int perm, int mode);
+	RList* /*RIODesc* */(*open_many)(RIO *io, const char *, int perm, int mode);
+	int(*read)(RIO *io, RIODesc *fd, ut8 *buf, int count);
```

Comment:
???? they were alright as they were before.

Reply:
that regex went too far

Reply:
:D :D i guess so